packages/lesswrong/components/common/SubscribedUsersFeed.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Components, registerComponent } from "../../lib/vulcan-lib/components";
+
+const styles = (theme: ThemeType): JssStyles => ({
+})
```

Comment:
nit: we should avoid explicitly annotating with `JssStyles` as a return type to avoid breaking type inference in `classes: ClassesType<typeof styles>`.  Maybe there's a way to configure an eslint rule for that (which doesn't interfere with other return-type annotations)...